docs/detailed-push-process.md

```diff
@@ -5,53 +5,49 @@ But while doing so, it makes sure those components are deployed correctly and wo
+The result of a successful staging process is a new image pushed to the Registry component of Epinio.
+
+This component is installed as part of the `epinio install` command and it is where the application images are stored. This is to make the setup easier (by not having to configure an external registry) and staging faster (by keeping all image transferring local to the cluster).
 There is not much to tell about it but if you want to look at how the registry is installed, have a look at the helm chart here:
 https://github.com/epinio/epinio/tree/main/assets/container-registry/chart/container-registry
 
-## 4. Creation of the Application Kubernetes Resources
+## 7. Run
 
 To run a workload on Kubernetes having a container image is not enough. You need at least a Pod running with at least one container running that image.
-The Tekton pipeline that created the image also created a [Deployment](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/) for your application.
-The deployment is set to use the generated image.
 
-That is enough to get a Pod running but you still don't have access to your application from outside the cluster. When you installed Epinio, it created and
-registered a [Kubernetes webhook](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/) which monitors the workloads namespace for new apps.
-When a new application appears, it creates a [Service](https://kubernetes.io/docs/concepts/services-networking/service/) and an [Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/) for it.
-You can read how these work in Kubernetes following the provided links but if you have to know one thing is that Ingress is the thing that describes how a request that uses the Application's url is routed to the application
-container. In Kubernetes, the thing that reads that description and implements the routing is called an Ingress Controller. Such an Ingress Controller is provided by [Traefik](https://doc.traefik.io/traefik/providers/kubernetes-ingress/).
+The last step of the staging Tekton pipeline is creating the runtime Kubernetes resources that are needed to make your application available to the users outside the Kubernetes cluster. The most important resources that are created are a [Deployment](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/), a [Service](https://kubernetes.io/docs/concepts/services-networking/service/) and an [Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/) resource. The Deployment is using the image that was pushed as part of the staging step (see arrows 6 and 8 on the image).
```

Comment:
`is creating` :arrow_right: `creates`

Question: `The most important resources are ...` implies that there are resources beyond Deployment, Service, and Ingress. What are these resources ?

Reply:
the cert-manager is creating a certificate for the application. I didn't want to describe that here to keep the focus on how the application becomes accessible, maybe it deserves another section to explain how certs work. Maybe on another PR?

I can't think of any other resources that are created, I hope I'm not missing something.

Reply:
yes, that is ok